Plugins/Creature/NWScript/nwnx_creature.nss

```diff
@@ -685,61 +685,69 @@ int NWNX_Creature_GetCasterLevelOverride(object oCreature, int nClass);
 /// @param oCreature The creature object.
 void NWNX_Creature_JumpToLimbo(object oCreature);
 
-/// @brief Sets the critical hit multiplier modifier for the creature
+/// @brief Sets the critical hit multiplier modifier for the Creature
 /// @param oCreature The target creature
 /// @param nModifier The modifier to apply
 /// @param nHand 0 for all attacks, 1 for Mainhand, 2 for Offhand
 /// @param bPersist Whether the modifier should persist to .bic file if applicable
-/// @note Persistence is activated each server reset by first use of either 'SetCriticalMultiplier*' functions. Recommended to trigger on a dummy target OnModuleLoad to enable persistence.
-void NWNX_Creature_SetCriticalMultiplierModifier(object oCreature, int nModifier, int nHand = 0, int bPersist = FALSE);
+/// @param nBaseItem Applies the.modifier only when the attack used this baseitem. -1 for 'Unarmed', '-2' for 'all'
```

Comment:
The game and nwnx generally use BASE_ITEM_GLOVES for unarmed.

Reply:
Swapped -1 out for Gloves constants, and changed the "all" from -2 to -1 accordingly.